changelog/dmd.deprecate-case-variables.dd

```diff
@@ -0,0 +1,44 @@
+Using run-time variables as switch cases has been deprecated
+
+It's a feature that's not commonly used, and it goes against normal expectations of a switch statement.
+When using run-time variables:
+
+- Case statements can contain duplicates
```

Comment:
And the alternative doesn't have duplicates?

Reply:
Yes, it's checked at compile time.

Reply:
```
int x;
if (x == 3) {

}

// ...
if (x == 3) {


}
```

Reply:
I thought 'alternative' was referring to compile-time case statements.

> And the alternative doesn't have duplicates?

It can have duplicates, but there's no expectation that consecutive if-statements are mutually exclusive.